src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -159,32 +167,34 @@ class FilterOptionsConfig {
     }
     // previousOptions will be null if there were no previousOptions in persistentStorage
     const previousOptions = config.previousOptions;
-    this.options = this.setSelectedOptions(this.options, previousOptions);
+    this.options = this.getSelectedOptions(this.options, previousOptions);
   }
 
   /**
-   * Sets selected options on load based on options stored in persistent storage and options with selected: true.
-   * If no previous options were stored in persistentStorage, default to options marked
-   * as selected. If multiple options are marked as selected for 'singleoption', only the
-   * first should be selected.
-   * @param {Array<Object>} options
-   * @param {Array<string>} previousOptions
-   * @returns {Array<Object>}
+   * Returns a list of options with `selected` determined by initialOptions and
+   * optionsOverrides. optionsOverrides take precedence over initialOptions. If the
+   * control is singleoption and `selected` appears more than once in either
+   * initialOptions or previousOptions then the first instance is used.
```

Comment:
I think `previousOptions` should be `optionsOverrides` here

Reply:
Fixed!